workflows/quickstart/index.js

```diff
@@ -0,0 +1,86 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+/**
+ * Sleeps the process N number of milliseconds.
+ * @param {Number} ms The number of milliseconds to sleep.
+ */
+function sleep(ms) {
+  return new Promise(resolve => {
+    setTimeout(resolve, ms);
+  });
+}
+
+/**
+ * Executes a Workflow and waits for the results with exponential backoff.
+ * @param {string} projectId The Google Cloud Project containing the workflow
+ * @param {string} location The workflow location
+ * @param {string} workflow The workflow name
+ */
+const main = async (
+  projectId = process.env.GOOGLE_CLOUD_PROJECT,
+  location = 'us-central1',
+  workflow = 'myFirstWorkflow'
+) => {
+  if (!projectId)
+    return console.error('ERROR: GOOGLE_CLOUD_PROJECT is required.');
+
+  // [START workflows_api_quickstart]
+  const {ExecutionsClient} = require('@google-cloud/workflows');
+  const client = new ExecutionsClient();
+
+  // Execute workflow
+  try {
+    const createExecutionRes = await client.createExecution({
+      parent: client.workflowPath(projectId, location, workflow),
```

Comment:
nit: save parent as a variable

Reply:
There's no reason to add a new variable/const, as the expression is not re-used.

Reply:
Right, but it may be more readable split apart. Preference is up to you.
```suggestion
      const parent = client.workflowPath(projectId, location, workflow);
      const createExecutionRes = await client.createExecution({parent});
```

Reply:
Yes, it could be argued either way. Less variables on the stack / in autocompletion results one way.

Reply:
Since this is a quickstart the emphasis is not on readability to the same degree as a snippet, however the (often forgotten) rubric guideline is to avoid nesting complex initialization logic inside objects.